### ISSUE TITLE
Disable Audit for a Table it remains tracking for delete.

### ISSUE THREAD

**Issue by nsimao** [thepirat000/Audit.NET#11]
Hello,

I used this code for disable audit for some tables:

```
            Audit.EntityFramework.Configuration.Setup()
                            .ForContext<MyContext>(config => config
                            .IncludeEntityObjects(false)
                            .AuditEventType("{context}:{database}"))
                            .UseOptOut()
                            .IgnoreAny(x => x.Name == "LogCmdClient" || x.Name == "BlackListBatch" || x.Name ==          "BlackListSQL" || x.Name == "ReleaseFiles");

```
But I noticed that when I delete something generates an occurrence in my audit log.

**Comment by thepirat000** [comment]
Are you using .edmx for the EF mapping?

**Comment by thepirat000** [comment]
This was fixed on version 4.6.0, you should be receiving the expected entity type. Please use the latest package.
